arch/sim/src/Makefile

```diff
@@ -125,10 +125,14 @@ ifeq ($(CONFIG_RTC_DRIVER),y)
   CSRCS += up_rtc.c
 endif
 
-ifeq ($(CONFIG_NX_LCDDRIVER),y)
+ifeq ($(CONFIG_SIM_LCDDRIVER),y)
   CSRCS += up_lcd.c
 else
-  CSRCS += up_framebuffer.c
+  ifeq ($(CONFIG_SIM_FRAMEBUFFER),y)
```

Comment:
change to "else ifeq ($(CONFIG_SIM_FRAMEBUFFER),y)"?

Reply:
if you don't have other observations can be leave it like this for now? I'm juggling various branches and I have to go back and rebase changes each time. I already have ready the putarea() PR waiting for this one.

Reply:
Sure.